### PR TITLE
Accessibility: correctly point labels to their respective fields

### PR DESCRIPTION
**Pull request by kaelig**
This fixes an issue where clicking on the label of a field wouldn't focus said field.

### Steps to reproduce:

1. Go to https://app.monicahq.com/settings
2. Click "Currency"

Expected: the "Currency" select is focused
Actual: the "Layout" select is focused

----

Note: this pull request won't fix absolutely all accessibility issues with forms but it's a start!

### CODE REVIEW

File: resources/views/journal/add.blade.php [hunk 1]
```diff
@@ -38,13 +38,13 @@
 
               {{-- Optional title --}}
               <div class="form-group">
-                <label for="title">{{ trans('journal.journal_add_title') }}</label>
-                <input type="text" class="form-control" name="title" autofocus>
+                <label for="feld-title">{{ trans('journal.journal_add_title') }}</label>
```
Comment (BranTurner) [inline resources/views/journal/add.blade.php]: Typo?

Comment (kaelig) [inline resources/views/journal/add.blade.php]: Good catch, thank you! Fixed :)

**Comment by kaelig** [comment]
@BranTurner I fixed a few merge conflicts, let me know if I need to do anything else!

**Comment by djaiss** [comment]
Thank so much for caring about the project and for this pull request.
The PR looks awesome, I'll merge it right away.

**Comment by kaelig** [comment]
Merci @djaiss!

**Pull request merged**